test/unit/jest/jest-setup-global.js

```diff
@@ -22,5 +30,27 @@ navigator.credentials = {
   }
 };
 
+// patch jquery 3 visible pseudos selector issue in jsdom
+// https://github.com/jsdom/jsdom/issues/1048#issuecomment-401599392
+// https://github.com/jsdom/jsdom/issues/1048#issuecomment-595961496
+window.Element.prototype.getClientRects = function() {
```

Comment:
Is there any workaround to avoiding using jquery to do the check for `:visible` instead of having to polyfill this behavior for jsdom?

Reply:
We would need to upate code within Gen 1 to not use the visible selector, [this](https://github.com/okta/okta-signin-widget/blob/master/src/v1/views/shared/Footer.js#L77) is one example where we are using the visible selector, but it is also used by JQuery qtip.